Makefile

```diff
@@ -50,16 +34,11 @@ build/.%.done: docker/Dockerfile.%
 	touch $@
 
 build/.flux.done: build/fluxd build/kubectl
-build/.flux-service.done: build/fluxsvc build/migrations.tar
 
-build/fluxd: $(FLUXD_DEPS)
+build/fluxd: vendor/
```

Comment:
What was wrong with `$(FLUXD_DEPS)`?

Reply:
It called godeps: `FLUXD_DEPS:=$(call godeps,./cmd/fluxd)`. Something I didn't notice when I did that particular migration.

Reply:
Oh is `godeps` not a thing any more

Reply:
Wait, yep I'm wrong. Derp.

Reply:
Do we vendor the other bits and pieces into vendor/ now?

Reply:
Which other bits and pieces are you referring to?